services/export/tests/functional/transactionsExport.test.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	standardizeIntervalFromParams,
+	getExcelFilenameFromParams,
+	getExcelFileUrlFromParams,
+} = require('../../shared/transactionsExport');
+
+const { interval } = require('../constants/csvExport');
+
+const {
+	STANDARDIZED_INTERVAL,
+	EXCEL_EXPORT_FILENAME,
+	EXCEL_FILE_URL,
+} = require('../../shared/regex');
+
+const config = require('../../config');
+
+describe('Excel export utils', () => {
+	const address = 'lskeqretdgm6855pqnnz69ahpojk5yxfsv2am34et';
+	const publicKey = 'b7fdfc991c52ad6646159506a8326d4203c868bd3f16b8043c8e4e034346e581';
+	const excelFilenameExtension = '.xlsx';
+	const excelFileUrlBeginsWith = '/api/v3/export/';
+
+	describe('Test standardizeIntervalFromParams method', () => {
+		it(`should return interval in standard format: '${config.excel.dateFormat}:${config.excel.dateFormat}'`, async () => {
+			const result = await standardizeIntervalFromParams({ interval: interval.startEnd });
+			expect(typeof result).toBe('string');
+			expect(result.length).toBe((2 * config.excel.dateFormat.length) + 1);
+			expect(result).toMatch(STANDARDIZED_INTERVAL);
+		});
+
+		it('should return standardized interval when both start and end date supplied', async () => {
+			const result = await standardizeIntervalFromParams({ interval: interval.startEnd });
+			expect(typeof result).toBe('string');
+			expect(result.length).toBe((2 * config.excel.dateFormat.length) + 1);
+			expect(result).toMatch(STANDARDIZED_INTERVAL);
+		});
```

Comment:
Looks duplicate, we can safely remove one.

```suggestion
		it(`should return standardized interval when both start and end date supplied`, async () => {
			const result = await standardizeIntervalFromParams({ interval: interval.startEnd });
			expect(typeof result).toBe('string');
			expect(result.length).toBe((2 * config.excel.dateFormat.length) + 1);
			expect(result).toMatch(STANDARDIZED_INTERVAL);
		});
```